src/main/resources/templates/budget.html

```diff
@@ -12,27 +12,36 @@
 <body>​
     <h1 class="container">Add Monthly Budget</h1>​
     <br>
-    <div class="container">
+    <div class="container row">
         <form>
-            <label for="month">Month:</label>
-            <input type="text" id="month" name="month" style="width: 300px;" class="form-control"><br><br><br>

```

Comment:
It looks like the reviewer is removing multiple br tags, and using CSS instead.  This is a good idea, as the layout is more flexible.  We often used multiple br tags in the days before css.  Now, we use margin to do the same thing.  With different CSS for different devices (monitor, tablet, phone, and wear), having a css style margin is more flexible than hardcoded BRs.